src/components/application_manager/rpc_plugins/app_service_rpc_plugin/src/app_service_rpc_plugin.cc

```diff
@@ -49,7 +49,9 @@ bool AppServiceRpcPlugin::Init(
     application_manager::ApplicationManager& app_manager,
     application_manager::rpc_service::RPCService& rpc_service,
     application_manager::HMICapabilities& hmi_capabilities,
-    policy::PolicyHandlerInterface& policy_handler) {
+    policy::PolicyHandlerInterface& policy_handler,
+    resumption::LastState& last_state) {
+  UNUSED(last_state);
```

Comment:
@ShobhitAd Please take a look at the previous comment's answer (https://github.com/smartdevicelink/sdl_core/pull/2984#discussion_r318022619). We can't move these changes into separate PR, because we need these changes in the scope of this feature.